lib/node/http.js

```diff
@@ -187,12 +189,16 @@ class HTTP extends Server {
     // Bulk read UTXOs
     this.post('/coin/address', async (req, res) => {
       const valid = Validator.fromRequest(req);
-      const address = valid.array('addresses');
+      const addressStrings = valid.array('addresses');
 
-      enforce(address, 'Address is required.');
+      enforce(addressStrings, 'Addresses is required.');
+      addressStrings.forEach(addrString => {
```

Comment:
Instead of using a `forEach` here, how about using a `for` loop and building the array of `Address` objects iteratively. Then we can get rid of the `map` call on line 200. `hsd` is built for speed and `forEach` and `map` are much slower than traditional `for` loops.